lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -538,7 +538,7 @@ def billing_name(payment_source, options)
           elsif options[:billing_address] && options[:billing_address][:name].present?
             options[:billing_address][:name]
           end
-        name ? I18n.transliterate(name || '')[0..29] : nil
+        name ? clean_emoji_emoticons_and_accent(name)[0..29] : nil
```

Comment:
I guess it doesn't harm to do this for orbital since their sandbox is not the most accurate anyways